src/components/SummaryManagement/SummaryManagement.jsx

```diff
@@ -109,7 +102,7 @@ class SummaryManagement extends Component {
     this.setState({
       deleteSummaryGroupPopupOpen: false,
     });
-    this.fetchSummaryGroup();
+    //this.fetchSummaryGroup();
```

Comment:
Do not forget to remove this comment line.

Reply:
I removed some duplicate functions, especially some functions related to Axios.get(), if there is still some duplicate code or could decouple it, please let me know. Also, update the crud for summary group  and  create and delete for summary receiver, please check, thanks